packaging/fedora/snapd.spec

```diff
@@ -500,7 +507,7 @@ autoreconf --force --install --verbose
 # selinux support is not yet available, for now just disable apparmor
 # FIXME: add --enable-caps-over-setuid as soon as possible (setuid discouraged!)
 %configure \
-    --disable-apparmor \
+    %{?with_apparmor:--enable-apparmor}%{!?with_apparmor:--disable-apparmor} \
```

Comment:
Brave new world :D

Reply:
Well, not really. It has no effect and is functionally useless, as it will be disabled unconditionally.